### ISSUE TITLE
Support for specifying Pester version to download

### ISSUE THREAD

**Issue by jakauppila** [test-kitchen/kitchen-pester#27]
It would be great to be able to specify a Pester version to download so a project could be stuck to a major so tests do not break with a major release (such as v4 which requires some changes from v3).

https://github.com/pester/Pester/wiki/Migrating-from-Pester-3-to-Pester-4

```
verifier:
  name: pester
    version: 3
```

This should be straightforward enough when utilizing `Install-Module`, it would just depend on what exactly you would want to allow for version specifications (just major, major.minor, specific?). I would really only be looking to stick to a particular major, so the following works:

`Install-Module -Name pester -MinimumVersion 3.0 -MaximumVersion 4.0`

**Comment by gaelcolas** [comment]
#64 will let you splat the parameters to `Install-Module -Name Pester`.
In the kitchen yaml config it will look like this:
```Yaml
  pester_install:
    RequiredVersion: 4.10.1
    Force: true
    AllowPrerelease: true
```

**Comment by gaelcolas** [comment]
This can be closed as #64 has been merged.

**Comment by gep13** [comment]
@gaelcolas thanks.
